modules/openapi-generator/src/main/resources/typescript/http/http.mustache

```diff
@@ -317,9 +317,9 @@ export function wrapHttpLibrary(promiseHttpLibrary: PromiseHttpLibrary): HttpLib
 
 export class HttpInfo<T> extends ResponseContext {
     public constructor(
-        public httpStatusCode: number,
-        public headers: Headers,
-        public body: ResponseBody,
+        httpStatusCode: number,
```

Comment:
i would suggest to add `override public` to make it easy to understand that these are members. wdyt?

Reply:
given they're passed on to the super via the constructor anyway, is that not doubled up?

Reply:
```ts
class X {
    constructor(public member: string) {}
}

class Y extends X {
    constructor(public override member: string) {
        super(member)
    }
}
```
yields:
```js

class X {
    constructor(public member: string) {}
}

class Y extends X {
    constructor(public override member: string) {
        super(member)
    }
}
.JS
.D.TS
Errors
Logs
Plugins
"use strict";
class X {
    constructor(member) {
        this.member = member;
    }
}
class Y extends X {
    constructor(member) {
        super(member);
        this.member = member;
    }
}
```
([playground link](https://www.typescriptlang.org/play/?target=9&noImplicitOverride=true&ssl=9&ssc=2&pln=1&pc=1#code/MYGwhgzhAEAa0G8CwAoa7rAPYDsIBcAnAV2Hy0IAoAHYgIxAEthoBbAU1bvcIC5oChRjgDmASkQBfVNJSpQkGAE1o7AB752OACYx4yNBmx4ipclVoNm0LADceQ7ezadufAUWHjEqDH4HE1DyUHFw8Yr4YspJAA))


```ts
class X {
    constructor(public member: string) {}
}

class Y extends X {
    constructor(member: string) {
        super(member)
    }
}
```
yields:
```js
"use strict";
class X {
    constructor(member) {
        this.member = member;
    }
}
class Y extends X {
    constructor(member) {
        super(member);
    }
}
```
Diff is:
```diff
- this.member = member;
```

which makes more sense to me. But if you feel strongly I can make these `public override` instead.

Reply:
fair point